conans/__init__.py

```diff
@@ -1,5 +1,7 @@
+import importlib.metadata
+
 CHECKSUM_DEPLOY = "checksum_deploy"  # Only when v2
 REVISIONS = "revisions"  # Only when enabled in config, not by default look at server_launcher.py
 OAUTH_TOKEN = "oauth_token"
 
-__version__ = '2.0.15-dev'
+__version__ = importlib.metadata.version("conan")
```

Comment:
I think this is only Python 3.8, we might need a bit of duplication until we move to 3.8